RFCs/Concepts_Redesign.md

```diff
@@ -0,0 +1,147 @@
+# Concepts redesign
+
+
+## Abstract
+
+This RFC proposes a simpler concept syntax, one that cuts right to the point of
+what a concept specifies.
+
+
+## Motivation
+
+The current concepts syntax is difficult to grok as it requires mentally
+evaluating Nim code. The concepts implementation is also currently very buggy
+and would require significant effort to make reliable.
```

Comment:
Probably a similar effort required to make the concepts this RFC proposes reliable. In fact most if not all cases these new concepts cover are reliable with the current concepts.